server/repository/src/migrations/version.rs

```diff
@@ -90,7 +90,7 @@ impl PartialOrd for Version {
         }
 
         if self.minor != other.minor {
-            return Some(self.major.cmp(&other.minor));
+            return Some(self.minor.cmp(&other.minor));
```

Comment:
Minor update was failing because of typo(?)

Reply:
Can you add this suggestion please (this should have been caught in a test, but tset was wrong) https://github.com/openmsupply/open-msupply/compare/1948-New-Field-mode-in-Invoice-table-...1948-suggestion